src/Table/index.jsx

```diff
@@ -1,16 +1,110 @@
 import React from 'react';
 import classNames from 'classnames';
+import FontAwesomeStyles from 'font-awesome/css/font-awesome.min.css';
+import isRequiredIf from 'react-proptype-conditional-require';
 import PropTypes from 'prop-types';
 
 import styles from './Table.scss';
+import Button from '../Button';
 
 class Table extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = props.tableSortable ?
```

Comment:
If the table is sortable, set the appropriate state variable to the defaultSortedColumn and defaultSortDirection; otherwise, leave both empty.